.storybook/main.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  stories: ['../packages/**/*.stories.mdx', '../packages/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@whitespace/storybook-addon-html',
+  ],
+  webpackFinal: async (config) => {
+    config.plugins.push(new MiniCssExtractPlugin()),
+      // add SCSS support for CSS Modules
+      config.module.rules.push(
+        {
+          test: /\.svg$/i,
+          use: [
+            {
+              loader: 'file-loader',
+              options: {
+                name: '[name].[ext]',
+                outputPath: 'images/',
+                publicPath: 'images/',
+              },
+            },
+            'url-loader',
+          ],
+        },
+        {
+          test: /\.(woff|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
+          use: [
+            {
+              loader: 'file-loader',
+              options: {
+                name: '[name].[ext]',
+                outputPath: 'fonts/',
+                publicPath: 'fonts/',
+              },
+            },
+            'url-loader?limit=100000',
+          ],
+          include: path.resolve(__dirname, '../'),
+        },
+        {
+          test: /\.scss$/,
+          use: [MiniCssExtractPlugin.loader, 'css-loader?url=false', 'sass-loader'],
+          include: path.resolve(__dirname, '../'),
+        }
+      );
+
+    return config;
```

Comment:
In the next iteration, we could alias these core paths that reference `dist`: https://github.com/CMSgov/design-system/blob/master/packages/ds-healthcare-gov/src/styles/index.scss#L12. That way, when we're live editing core styles and viewing the child design system storybook, we don't have to run `yarn build` before we see those style changes. This is the problem with the CDS migration that I wanted to fix by moving our build scripts to use webpack, but that is a big undertaking. Since we want to use Storybook for local development anyway, having that functionality here means we can put off adding that functionality to the docs site generator for quite a while.

Reply:
https://webpack.js.org/configuration/resolve/#resolvealias

Reply:
Yep! I'll include this with the child system storybook setup because it seems most relevant to that work.